apps/coordinator/src/components/ScriptExplorer/Transaction.jsx

```diff
@@ -34,14 +42,30 @@ class Transaction extends React.Component {
 
   buildSignedTransaction = () => {
     const { network, inputs, outputs, signatureImporters } = this.props;
-    return signedMultisigTransaction(
+    const args = {
       network,
-      inputs,
-      outputs,
-      Object.values(signatureImporters).map(
-        (signatureImporter) => signatureImporter.signature,
-      ),
-    );
+      inputs: inputs.map(convertLegacyInput),
+      outputs: outputs.map(convertLegacyOutput),
+    };
+    const psbt = getUnsignedMultisigPsbtV0(args);
+    let partiallySignedTransaction = psbt.toBase64();
+    for (const signatureImporter of Object.values(signatureImporters)) {
+      partiallySignedTransaction = addSignaturesToPSBT(
+        Network.REGTEST,
+        partiallySignedTransaction,
+        signatureImporter.publicKeys.map((pubkey) =>
+          Buffer.from(pubkey, "hex"),
+        ),
+        signatureImporter.signature.map((signature) =>
+          Buffer.from(signature, "hex"),
+        ),
+      );
+    }
+
+    return Psbt.fromBase64(partiallySignedTransaction)
+      .finalizeAllInputs()
+      .extractTransaction()
+      .toHex();
```

Comment:
In the newer versions of these functions, I opted to make them less "magical" and extract some of the logic that was happening under the hood. Previously, `signedMultisigTransaction` would take inputs, outputs, and signatures and then create a PSBT and then add the various signatures for you. IMO this abstracts away the PSBT saga too much and then makes the functions tightly coupled to a specific API and expected saga flow. 

The new approach is to expect the consumer of these functions to just add the signature it needs to add to a given PSBT (`addSignaturesToPsbt` already existed and didn't need to be upgraded). So this change first creates the PSBT (ideally we're just passing around PSBTs at this point so this step can be skipped), then finds each pubkey and signature to add to the psbt. the legacy `signedMultisigTransaction` did this all "automatically"

Additionally, since this component method `buildSignedTransaction` is already building the finalized transaction, it can just return the raw hex.